cron/reset.py

```
@@ -0,0 +1,36 @@
+
+with image.imports():
+    from rush import quota, throttle
+    from rush.limiters import periodic
+    from rush.stores import redis as redis_store
+
+
+@app.function(
+    schedule=modal.Cron("0 8 * * *"),  # PST time. Every day at 8:00
+    retries=modal.Retries(max_retries=3),
+)
+def clear_limit():
```

Ruff

Missing return type annotation for public function `clear_limit`

(ANN201)